### ISSUE TITLE
Questionnaire: index by solr the question when it is imported dynamically

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/catalogue#928]

**Comment by ribeiroct** [comment]
@bastiao :
I am not sure this is valid. I was looking today at the way things are indexed, to try and solve #1020, and just checked that the questionnaire indexation is done by signals sent by the Questionnaire, Questionset and Question models after saving alterations. So even though this is done only 10 minutes after (so we dont spam solr) it should be worked in dynamically imported question.

**Comment by bastiao** [comment]
@ribeiroct yes, you're right. it is not an issue anymore. It is only take 10minutes after modification in questionnaire. 

but look #549 I think that maybe there are related issues that should be solved at solr.
